src/impl/realm_coordinator.hpp

```diff
@@ -21,6 +21,8 @@
 
 #include "shared_realm.hpp"
 
+#include <realm/version_id.hpp>
```

Comment:
I guess Xcode resolves these differently, since commenting this out still compiled.